src/Catalyst.Core.Modules.Consensus/IO/Observers/CandidateDeltaObserver.cs

```diff
@@ -52,25 +56,33 @@ public override void HandleBroadcast(IObserverDto<ProtocolMessage> messageDto)
             {
                 Logger.Verbose("received {message} from {port}", messageDto.Payload.CorrelationId.ToCorrelationId(),
                     messageDto.Payload.PeerId.Port);
+                
+                // @TODO here we use the protobuff message to parse rather than using the CandidateDeltaBroadcastDao
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)